src/redux/books/books.js

```diff
@@ -0,0 +1,38 @@
+import * as actions from '../actionTypes';
+
+// initial state
+const initialState = [];
+
+// reducer
+export default function booksReducer(state = initialState, action = {}) {
+  switch (action.type) {
+    case actions.REMOVE_BOOK: return [
+      ...state.filter((book) => book.id === action.payload.id),
+    ];
+    case actions.ADD_BOOK: return [
+      ...state.push(action.payload),
+    ];
+    default: return state;
+  }
+}
+
+// action creators
+export function removeBook() {
+  return {
+    type: actions.REMOVE_BOOK,
+    payload: {
+      id: 1,
+    },
+  };
+}
+
+export function addBook() {
+  return {
+    type: actions.ADD_BOOK,
+    payload: {
+      id: 1,
+      title: '',
+      author: '',
+    },
+  };
```

Comment:
- Nice Job for implementing the code. Kindly make sure `Add` and` Remove` work when we run the App
- 
![Capture](https://user-images.githubusercontent.com/63972670/208664421-fb233529-8dd4-4806-818d-bb37b2456a03.PNG)